CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [0.6.0] - 2023-05-17
+BREAKING CHANGES
```

Comment:
Not all of them are breaking changes. Look how we did it cartridge-java 
![image](https://github.com/tarantool/cartridge-java-testcontainers/assets/22258443/240679f9-96bb-4b4e-85ec-c6450a09b3e5)